torch/_inductor/ir.py

```diff
@@ -5972,13 +5988,26 @@ def create(cls, kernel, *args, **kwargs):
             V.graph.fake_mode if kernel not in fake_incorrect_kernels else nullcontext()  # type: ignore[assignment]
         )
         with context:
+
+            if (
+                kernel == torch.ops.fsdp.all_gather_copy_in.default
+                or kernel == torch.ops.fsdp.split_with_sizes_copy.default
+                or kernel == torch.ops.fsdp.chunk_cat.default
+                or kernel == torch.ops.fsdp.read_out.default
+            ):
+                process_kernel_seperate = True
```

Comment:
in general I wonder what if these ops are from FSDP2 rather than Simple FSDP -- what would it cause?